testsuite/forge/src/backend/k8s/chaos.rs

```diff
@@ -314,4 +263,26 @@ impl K8sSwarm {
 
         Ok(())
     }
+
+    /// Returns the instance labels for the given peers
+    /// as a string (separated by commas).
+    fn get_instance_labels(&self, peers: &[PeerId]) -> String {
+        peers
+            .iter()
+            .map(|node| self.get_node_name(node))
+            .collect::<Vec<_>>()
+            .join(",")
+    }
+
+    /// Returns the name of the node associated with the given account address
+    fn get_node_name(&self, node: &AccountAddress) -> &str {
+        if let Some(validator) = self.validator(*node) {
+            validator.name()
+        } else if let Some(fullnode) = self.full_node(*node) {
+            fullnode.name()
+        } else {
+            // TODO: should we throw an error here instead of failing silently?
```

Comment:
True, better to throw an error. Would save someone a lot of time.

Reply:
Agreed 😄 Will follow-up to unblock this!